tests/PropertyTypeTest.php

```diff
@@ -2790,7 +2790,7 @@ class Foo {
                     echo $a->foo;',
                 'error_message' => 'NullPropertyFetch',
             ],
-            'badArrayProperty' => [
+            'SKIPPED-badArrayProperty' => [ // PropertyTypeCoercion?
```

Comment:
Ignore this, it was failing previously and I thought `B` extended `A`, which does indeed report `PropertyTypeCoercion` already.